docs/advanced/swap.md

```diff
@@ -187,4 +187,36 @@ You may use then Etherscan to track your transaction and make sure it completes
 
 [https://goerli.etherscan.io/tx/0xedc80ebc89e6d719e617a50c6900c3dd5dc2f283e1b8c447b9065d7c8280484a`](https://goerli.etherscan.io/tx/0xedc80ebc89e6d719e617a50c6900c3dd5dc2f283e1b8c447b9065d7c8280484a)
 
+# Managing uncashed cheques
+For the Bee process, the final state of earning (g)BZZ is a swap-cheque. It is worthy to note, however, that a cheque is not yet generally accepted currency. A swap-cheque is, just like a real cheque, a promise to hand over money upon request. In real life, you would present the cheque to a bank. In blockchain life, we are presenting the swap-cheque to a smart-contract. Holding on to a swap-cheque is risky; it is possible that the owner of the chequebook has written written cheques, worth more than there is money in the chequebook contract. Because of this reason you want to cash out your cheques with a certain frequency. With the set of API endpoints, as offered by Bee, it is possible to develop a script which fully manages the uncashed cheques for you. As an example, we offer you a [very basic script](https://gist.github.com/ralph-pichler/3b5ccd7a5c5cd0500e6428752b37e975#file-cashout-sh), where you can manually cash out all cheques with a worth above a certain value. To use the script:
```

Comment:
i suggest not complicating the docs with the (g)BZZ details. describe in the introduction that the testnet is called Goerli, and the situation with gBZZ while running on the test net. then in the rest of the doc just call it BZZ, so that we don't need to readjust the docs later, when it graduates to the mainnet.